src/issue-filing/components/issue-filing-choice-group.tsx

```diff
@@ -33,6 +33,7 @@ export const IssueFilingChoiceGroup = NamedFC<IssueFilingChoiceGroupProps>('Issu
     return (
         <ChoiceGroup
             className={'issue-filing-choice-group'}
+            ariaLabelledBy="issue-filing"
```

Comment:
2 comments:
1. I think this should be extracted to a common `const` so it can be used on both places (where we "define" it and where we "use" it)
2. Is it `"issue-filing-title-id"` a better value for this id?

Reply:
I removed id from the actual title, but named the variable `issueFilingHeadingId`